fasm2bels/models/verilog_modeling.py

```diff
@@ -1043,17 +1043,18 @@ def decode_multi_bit_feature(self, feature):
         """
         Decodes a "multi-bit" fasm feature. If not present returns 0.
         """
-        value = 0
```

Comment:
So this new code is not right, because multibit features have two basic representations:
```
FEAT_A.B[3:0] = 4'b1111
```
or
```
FEAT_A.B[3] = 1
FEAT_A.B[2] = 1
FEAT_A.B[1] = 1
FEAT_A.B[0] = 1
```
Which are equivilent.

Reply:
Right. I've fixed that and added a test.